app.js

```diff
@@ -155,31 +163,33 @@ function LeftButton(route, navigator, index, navState) {
       if (index === 1) {
         backTitle = 'Home'
       }
-      return (
-        <TouchableOpacity
-          style={styles.backButton}
-          onPress={() => navigator.pop()}
-        >
-          <Icon style={styles.backButtonIcon} name='ios-arrow-back' />
-          <Text style={styles.backButtonText}>{backTitle}</Text>
-        </TouchableOpacity>
-      )
+      if (Platform.OS === 'android') {
+        return (
+          <TouchableOpacity
+            style={styles.backButton}
+            onPress={() => navigator.pop()}
+          >
+              <Icon style={styles.backButtonIcon} name='md-arrow-back' />
+          </TouchableOpacity>
```

Comment:
Don't Android apps (thinking Gmail) usually have some text next to the button? Phone is currently dead so can't check.